docker/azure-functions/release.sh

```diff
@@ -23,7 +23,10 @@ do
         --build-arg PYTHON_VERSION=$version \
         -t bentoml/azure-functions:$BENTOML_VERSION-py${version//.} \
         .
-    docker push bentoml/azure-functions:$BENTOML_VERSION-py${version//.}
+
+    if ! [[ -t 1 ]]; then
+      docker push bentoml/azure-functions:$BENTOML_VERSION-py${version//.}
```

Comment:
Just to clarify - if I run `./dev/release-docker-images.sh`, it will hit this line and push the image. if I run `docker/azure-functions/release.sh` by itself, it will skip this line?

Reply:
hmm this assumes that we release our images through CI (since we don't actually invoke it in a interactive session). If we run this manually that this doesn't work as expected.